docs/source/install/upgrades.rst

```diff
@@ -43,16 +43,22 @@ Upgrade Procedure
 
      sudo yum update $PKG_NAME
 
-3. Run the migration script (if any). See section below for |st2| 
-   version-specific migration scripts.
-
-4. Upgrade Mistral database.
+3. Upgrade Mistral database.
 
 .. sourcecode:: bash
 
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
```

Comment:
I think this is a long due change. If we agree these steps should be run on every install (not just upgrade), we can just put these in post-inst (forgot the rpm equivalent).